salad/cli.py

```diff
@@ -8,37 +9,34 @@
 from salad.terrains.everything import *
 
 BROWSER_CHOICES = [browser.lower()
-                   for browser in DesiredCapabilities.__dict__.keys()
+                   for browser in list(DesiredCapabilities.__dict__.keys())
                    if not browser.startswith('_')]
 BROWSER_CHOICES.sort()
 DEFAULT_BROWSER = 'firefox'
+DEFAULT_PLATFORM = 'Linux'
+PLATFORM_CHOICES = ['Linux', 'OS X 10.6', 'Windows XP', 'Windows 7',
+                    'Windows 8', 'Windows 8.1']
 
-class store_driver_and_version(argparse.Action):
-    drivers = BROWSER_CHOICES
-
-    def __call__(self, parser, namespace, values, option_string=None):
-        driver_info = values.split('-')
-        if driver_info[0] not in self.drivers:
-            args = {'driver': driver_info[0],
-                    'choices': ', '.join(map(repr, self.drivers))}
-            message = 'invalid choice: %(driver)r (choose from %(choices)s)'
-            raise argparse.ArgumentError(self, message % args)
-        setattr(namespace, self.dest, driver_info[0])
-        if len(driver_info) > 1:
-            setattr(namespace, 'version', driver_info[1])
-        if len(driver_info) > 2:
-            setattr(namespace, 'platform', driver_info[2].replace('_', ' '))
 
 def main(args=sys.argv[1:]):
     parser = argparse.ArgumentParser(prog="Salad",
                                      description=("BDD browswer-automation "
                                                   "made tasty."))
 
+    parser.add_argument('-V', action='store_true', default=False,
```

Comment:
would prefer this as `--version`, so it's not confused with the more common `-v` == verbose